libres/lib/analysis/std_enkf.cpp

```diff
@@ -96,10 +98,56 @@ static UTIL_SAFE_CAST_FUNCTION_CONST(std_enkf_data, STD_ENKF_TYPE_ID)
     return data->truncation;
 }
 
+/*
+  The std_enkf module originally used two boolean flags use_EE and use_GE to
```

Comment:
For us newbies - what are `EE` and `GE`?

Reply:
Hehe - I am glad you that ask that question .... frankly don't know fully myself either.

It is about choosing the algorithm for (lowrank)inversion of the covariance matrix - which in general do not have. The `use_EE` flag is whether we should use the approximation `C = EE^T`. If the `use_EE` flag is false the (pseudo)inversion is done using the S matrix - without ever constructing or using `E`.

The difference between `use_GE` is a formulation where `E` is used - but not the covariance estimator `EE^T`.

Would have been healthy to go properly through this again.



The purpose of this PR is alignment and code reuse between the old `std_enkf` implementation and the newer `ies_enkf`.

Reply:
That makes a little bit of sense, but I am not yet that comfortable with the math. Thanks!

I think what we want is to document the different types in `ies_inversion_type` and link them to literature.

I have created an issue that I've called [Questions for Geir Evensen](https://github.com/equinor/ert/issues/2362) and have now added this question as well.